catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcTableOperations.java

```diff
@@ -124,10 +124,36 @@ public List<String> listTables(String databaseName) throws NoSuchSchemaException
     }
   }
 
+  /**
+   * Get table information from the result set and attach it to the table builder, If the table is
+   * not found, it will throw a NoSuchTableException.
+   *
+   * @param resultSet The result set of the table
+   * @return The builder of the table to be returned
+   */
+  protected JdbcTable.Builder attachTableToBuilder(
```

Comment:
seems the difference is whether check schema , how about add an abstract method like `boolean supportsSchema()`?  if yes, check the schema, if not, do normal logic.

Reply:
`supportsSchema()` does not seem to be a very desirable name, we may name it `needToCheckSchemaName` or something similar.

Reply:
@mchades 
Please also take a look about it, thanks.